File: Suhyun/JS/문자열 다루기 기본/String_Control_Default.js
function solution(s) {
    let answer =true;
    let reg =/^[0-9]*$/;
    
    // isNaN은 1e22도 false 반환 
    
    if(reg.test(s)==false)
    {
     return false;
    }
    else if(s.length==4 || s.length==6){
         return true;
    }
    else{
      return false;
    }
}

File: Suhyun/Java/문자열 다루기 기본/String_Control_Default.java
public class String_Control_Default {
    public static void main(String[] args) {
        String s = "1234231";
        boolean answer =true;

        //숫자인지 아닌지 파악 
        if( s.matches("^[0-9]+$")==false )
        {
            System.out.println(false);
            //return false;
        }
        //길이가 4 혹은 6
        else if(4==s.length() && s.length()==6){
            System.out.println(answer);
            //return answer;
        }
        else {
            System.out.println(false);
            //return false;
        }
        
    }
    
}